{{cookiecutter.project_slug}}/compose/production/django/Dockerfile

```diff
@@ -7,31 +7,37 @@ RUN npm install && npm cache clean --force
 COPY . /app
 RUN npm run build
 
-# Python build stage
 {%- endif %}
-FROM python:3.8-slim-buster
 
-ENV PYTHONUNBUFFERED 1
+# Python build stage
+FROM python:3.8-slim-buster as python-build-stage
```

Comment:
The changes in this file look unrelated. Could you please make sure your pull request has only the appropriate change?